cpu/esp32/esp-now/esp_now_netif.c

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
Since this is also required for #9917, I'd rather move it to `sys/net/gnrc/netif/esp-now`.

Reply:
Also see https://github.com/RIOT-OS/RIOT/pull/9917#discussion_r244182732

Reply:
Do you mean the ingroup? Changed that to esp_now_gnrc in all the files now. Everything resides in the cpu/esp_common/esp-now now.

Reply:
No. Change the file name to `esp_now_gnrc.c` or something like that.

Reply:
But now that you mention it: where is the group `esp_now_gnrc` defined? Shouldn't it rather be in `drivers_esp32_esp_now` (which needs to be renamed, now that `esp_now` is in `esp_common`)?